dev/rpm/volta-updates.spec

```diff
@@ -0,0 +1,70 @@
+Name:           volta
+Version:        0.6.3
+Release:        1%{?dist}
+Summary:        The JavaScript Launcher ⚡
+
+License:        BSD 2-CLAUSE
+URL:            https://%{name}.sh
+Source0:        https://github.com/volta-cli/volta/archive/v%{version}.tar.gz
+
+# cargo is required, but installing from RPM is failing with libcrypto dep error
+# so you will have to install cargo manually to build this
```

Comment:
Are we documenting this in install docs as well?

Reply:
As above, @mikrostew probably has the most context on this, but I believe this comment is about an end user trying to use the RPM to build from source, not an install dep per se.

Reply:
Ahhhh, got it, makes sense. <3